.github/workflows/bwwl_operations.yml

```
@@ -12,7 +12,7 @@ on:
         required: false
         type: string
   schedule:
-  - cron: '0 0 1 * *'
+  - cron: '0 0 */14 * *'
```

Looks like every two weeks is [difficult](https://stackoverflow.com/questions/46109358/how-to-create-a-cron-expression-for-every-2-weeks) to nail down in cron. Let's go with every Monday at midnight.
```suggestion
  - cron: '0 0 * * 1'
```